src/frontend/Sider/Behandling/VedtakOgBeregning/Barnetilsyn/InnvilgeVedtak/InnvilgeBarnetilsyn.tsx

```diff
@@ -59,10 +67,34 @@ export const InnvilgeBarnetilsyn: React.FC<Props> = ({ lagretVedtak }) => {
         // eslint-disable-next-line
     }, [lagretInnvilgetVedtak]);
 
+    const lagreVedtak = (vedtaksRequest: InnvilgeVedtakForBarnetilsyn) => {
+        // eslint-disable-next-line no-console
+        console.log('Lagre', vedtaksRequest);
+    };
+
+    const handleSubmit = (form: FormState<InnvilgeVedtakForm>) => {
+        const vedtaksRequest: InnvilgeVedtakForBarnetilsyn = {
+            perioder: form.utgiftsperioder,
+            // begrunnelse: form.begrunnelse,
+            _type: VedtakType.InnvilgelseBarnetilsyn,
+            resultatType: BehandlingResultat.INNVILGET,
+        };
+        lagreVedtak(vedtaksRequest);
+        return form;
+    };
+
     return (
         // TODO: HandleSubmit
-        <Form>
-            <UtgiftsperiodeValg utgiftsperioderState={utgiftsperiodeState} />
+        <Form onSubmit={formState.onSubmit(handleSubmit)}>
```

Comment:
Fjerne TODO på handle submit?

Reply:
Evt. flytte til submit-funksjonen

Reply:
Fjernet den bare, er det vits å legge til TODO kommentarer på arbeid som er underveis nå? 
Pleier å legge til TODO kommentarer dersom det er noe som ikke kommer til å legges til nå fordi det er for tidlig, men etterhvert må det inn.

Reply:
Nei, ikke vits der du jobber nå. Tenker mer typ denne koden kommer jeg ikke til å røre før ...